pvlib/spectrum/mismatch.py

```diff
@@ -386,20 +382,18 @@ def spectral_factor_firstsolar(precipitable_water, airmass_absolute,
     * spectrum simulated on a plane normal to the sun
     * All other parameters fixed at G173 standard
 
-    From these simulated spectra, M is calculated using the known
+    From these simulated spectra, :math:`M` is calculated using the known
     quantum efficiency curves. Multiple linear regression is then
-    applied to fit Eq. 1 to determine the coefficients for each module.
-
-    Based on the PVLIB Matlab function ``pvl_FSspeccorr`` by Mitchell
-    Lee and Alex Panchula of First Solar, 2016 [2]_.
+    applied to fit Eq. 1 to determine the coefficients for each module. More
+    details on the model can be found in [2]_.
 
     Parameters
     ----------
     precipitable_water : numeric
         atmospheric precipitable water. [cm]
 
     airmass_absolute : numeric
-        absolute (pressure-adjusted) air mass. [unitless]
+        absolute (pressure-corrected) air mass. [unitless]
```

Comment:
```suggestion
        absolute (pressure-adjusted) air mass. [unitless]
```
There's nothing incorrect about the non-pressure-adjusted airmass.